docs/errors/E0717.md

```diff
@@ -0,0 +1,19 @@
+# E0717: import alias cannot use 'import type'
+
+
+The error message "import alias cannot use 'import type'" occurs when
+trying to use the 'import type' syntax with an alias.
+
+
+```typescript
+import type A = ns;
+```
+
+
+To fix this error, you need to use the regular 'import' syntax instead
+of 'import type' when using an alias.
+
+
+```typescript
+import { ns as A } from './my-module';
```

Comment:
Must fix: This is not the correct fix for the code. The correct fix would be to remove `type`:
```suggestion
import A = ns;
```